tests/lib/snaps/test-snapd-content-slot-empty-content-attr/meta/snap.yaml

```diff
@@ -0,0 +1,7 @@
+name: test-snapd-content-slot-empty-content-attr
+version: 1.0
+slots:
+    shared-content:
+        interface: content
+        read:
+            - /
```

Comment:
I will do a separate branch for this for master. When fixing it in 2.23 I got some strange errors and I do not want to hold up 2.23.1 futher.